controllers/syncer/csi_controller.go

```diff
@@ -46,6 +46,7 @@ const (
 	snapshotterContainerName             = "csi-snapshotter"
```

Comment:
can you add `volumeGroupContainerName` to [this case](https://github.com/IBM/ibm-block-csi-operator/blob/166783ed3bab275049decb4eb4f1d15681c729d3/controllers/syncer/csi_controller.go#L336)?

Reply:
Done